Libraries/Utilities/differ/transformsDiffer.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+'use strict';
+
+const transformsDiffer = function(one, two) {
+  return one !== two ||
+    !one || !two ||
+    one.length !== two.length ||
+    JSON.stringify(one) !== JSON.stringify(two);
```

Comment:
Why do we need to use `JSON.stringify` here?
Which kinda values we diff here?

Reply:
An array of objects (eg: `[{ scaleX: 0.5 }, { translateX: 100 }]`) where order matters.